src/site/content/en/blog/introducing-model-viewer/index.md

```diff
@@ -0,0 +1,139 @@
+description: |
+3D models are more relevant than ever. Retailers bring in-store shopping
+experiences to customers' homes. Museums let anyone see artifacts from anywhere.
+3D is difficult to do without a deep knowledge of 3D technologies or third-party
+hosting. <model-viewer> 1.0 makes these tasks as easy as writing HTML.
+date: 2020-08-20
+hero: hero.jpg
+alt: A 3d image of a shark.
+tags:
+  - blog
+  - 3d
+  - model-viewer
+---
+
+3D models are more relevant than ever. Retailers bring in-store shopping
+experiences to customers' homes. Museums are making 3D models of their artifacts
+available to everyone on the web. Unfortunately, it can be difficult to add a 3D
+model to a website in a way that provides a great user experience without a deep
+knowledge of 3D technologies or resorting to hosting 3D content on a third-party
+site. The `<model-viewer>` web component, [introduced in early
+2019](../model-viewer), seeks to
+make putting 3D models on the web as easy as writing a few lines of HTML. Since
+then, the team has been working to address feedback and requests from the
+community. The culmination of that work is `<model-viewer>` version 1.0.
+
+## What's new since last year?
+
+Version 1.0 includes built-in support for augmented reality (AR) on the web,
+improvements to speed and fidelity, and other frequently-requested features.
+
+### Augmented reality
+
+Viewing a 3D model on a blank canvas is great, but being able to view it in your
+space is even better. Now `<model-viewer>` supports [WebXR augmented
+reality](https://modelviewer.dev/examples/webxr.html0), for an
+entirely-within-the-browser 3D and AR experience on Chrome Android.
+
+```html
+<model-viewer src="ToyTrain.glb" ar ar-scale="auto" camera-controls alt="A 3D model of a wooden toy train">
+</model-viewer>
+```
+<iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://modelviewer.dev/webdotdev/webxr.html" frameborder="0" allowfullscreen></iframe>
+
+The above example shows the minimum required to use AR. The `<model-viewer>`
+component also allows you use HTML to customize the WebXR AR experience, as
+shown in [this example on the modelviewer.dev
+site](https://modelviewer.dev/examples/webxr.html).
+
+### Camera controls
+
+`<model-viewer>` now gives full control over the view's virtual camera (the
+perspective of the viewer). This includes the camera target, orbit (position
+relative to the model), and field of view. You can also enable auto-rotation and
+set limits on user interaction (e.g. maximum and minimum fields of view).
+
+### Annotations
+
+You can also annotate your models using HTML and CSS. This capability is often
+used to "attach" labels to parts of the model in a way that moves with the model
+as it's manipulated. The annotations are customizable, including their
+appearance and the extent to which they're hidden when they're positioned behind
+the model. Annotations also work in AR.
```

Comment:
I would say "not facing the viewer" rather than "positioned behind the model", just to be precise (we don't do physical occlusion).